tests/xharness/xharness.csproj

```diff
@@ -73,11 +73,7 @@
     <PackageReference Include="Mono.Options" Version="6.6.0.161" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="AppRunner.cs" />
```

Comment:
From these, we will need `CrashSnapshotReporterFactory`, `DeviceLogCapturerFactory` and things you will need for app bundling.

We won't really need Simulator/DeviceLoader factories and you will see about RunTasks as per our conversation offline